CMakeLists.txt

```diff
@@ -294,6 +274,10 @@ if(BUILD_TESTING)
   endif()
   find_package(Threads REQUIRED)
   target_link_libraries(ninja_test PRIVATE libninja libninja-re2c GTest::gtest Threads::Threads)
+  if (MSVC)
```

Comment:
Can this be WIN32 instead, since I do cross-compile using mingw64 on Linux

Reply:
Made the change